Source/AwsCommonRuntimeKit/mqtt/Mqtt5Types.swift

```diff
@@ -442,91 +466,135 @@ public enum ClientOperationQueueBehaviorType {
-public enum PayloadFormatIndicator: Int {
+public enum PayloadFormatIndicator {
 
     /// The payload is arbitrary binary data
-    case bytes = 0
+    case bytes
 
     /// The payload is a well-formed utf-8 string value.
-    case utf8 = 1
+    case utf8
 }
 
 extension PayloadFormatIndicator {
-    var nativeValue: aws_mqtt5_payload_format_indicator {
-        return aws_mqtt5_payload_format_indicator(rawValue: UInt32(self.rawValue))
+    var rawValue: aws_mqtt5_payload_format_indicator {
+        switch self {
+        case .bytes: return AWS_MQTT5_PFI_BYTES
+        case .utf8: return AWS_MQTT5_PFI_UTF8
+        }
+    }
+
+    /// Initializes Swift enum from native representation
+    init(_ cEnum: aws_mqtt5_payload_format_indicator) {
+        switch cEnum {
+        case AWS_MQTT5_PFI_BYTES:
+            self = .bytes
+        case AWS_MQTT5_PFI_UTF8:
+            self = .utf8
+        default:
+            fatalError("Unknown QoS Value")
+        }
     }
 }
 
 /// Configures how retained messages should be handled when subscribing with a topic filter that matches topics with
 /// associated retained messages.
 /// Enum values match `MQTT5 spec <https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901169>`_ encoding values.
-public enum RetainHandlingType: Int {
+public enum RetainHandlingType {
 
     /// The server should always send all retained messages on topics that match a subscription's filter.
-    case sendOnSubscribe = 0
+    case sendOnSubscribe
 
     /// The server should send retained messages on topics that match the subscription's filter, but only for the
     /// first matching subscription, per session.
-    case sendOnSubscribeIfNew = 1
+    case sendOnSubscribeIfNew
 
     /// Subscriptions must not trigger any retained message publishes from the server.
-    case dontSend = 2
+    case dontSend
 }
 
 extension RetainHandlingType {
-    var natvieValue: aws_mqtt5_retain_handling_type {
-        return aws_mqtt5_retain_handling_type(rawValue: UInt32(self.rawValue))
+    /// Returns the native representation of the Swift enum
+    var rawValue: aws_mqtt5_retain_handling_type {
+        switch self {
+        case .sendOnSubscribe: return AWS_MQTT5_RHT_SEND_ON_SUBSCRIBE
+        case .sendOnSubscribeIfNew: return AWS_MQTT5_RHT_SEND_ON_SUBSCRIBE_IF_NEW
+        case .dontSend: return AWS_MQTT5_RHT_DONT_SEND
+        }
     }
 }
 
 /// An enumeration that controls how the client applies topic aliasing to outbound publish packets.
 /// Topic alias behavior is described in `MQTT5 Topic Aliasing <https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901113>`_
-public enum OutboundTopicAliasBehaviorType: Int {
+public enum OutboundTopicAliasBehaviorType {
     /// Maps to Disabled.  This keeps the client from being broken (by default) if the broker
     /// topic aliasing implementation has a problem.
-    case `default` = 0
+    case `default`
```

Comment:
Would we consider to rename `default` to other name to avoid confusing between swift default keyword?

Reply:
changed it to `defaultBehavior`